pkg/api/api.go

```diff
@@ -47,8 +49,15 @@ const (
 var (
 	errInvalidNameOrAddress = errors.New("invalid name or bzz address")
 	errNoResolver           = errors.New("no resolver connected")
+	errInvalidPostageBatch  = errors.New("invalid postage batch id")
+
+	fallbackPostageBatch []byte
```

Comment:
This is a mutable shared variable between multiple goroutines and we cannot be sure that it will not be mutated, as it is passed to other components. I assume that it is an optimization to avoid allocations, but, I think that this optimization is not needed, and it is better to be safe.